### PR TITLE
created seed data for admin user

### PR DESCRIPTION
**Pull request by dannyvassallo**

### CODE REVIEW

**Comment by dannyvassallo** [comment]
Merging to move along.

File: db/seeds.rb [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
+    email:  "admin@example.com",
+    password:  "helloworld",
+    password_confirmation: "helloworld"
+  )
+user.skip_confirmation!
+user.save!
+
+# Create a list
+1.times do
+  @user = User.last
```
Comment (eliotsykes) [inline db/seeds.rb:14]: No need for @instance variable here, just use `user` that was assigned on line 4.

File: db/seeds.rb [hunk 2]
```diff
@@ -1,7 +1,32 @@
...
+user = User.new(
+    email:  "admin@example.com",
+    password:  "helloworld",
+    password_confirmation: "helloworld"
+  )
+user.skip_confirmation!
+user.save!
+
+# Create a list
+1.times do
```
Comment (eliotsykes) [inline db/seeds.rb:13]: `1.times` can be removed

File: db/seeds.rb [hunk 3]
```diff
@@ -1,7 +1,32 @@
...
+  @user = User.last
+  @user.lists.create!(
+    title: Faker::Lorem.sentence
+  )
+end
+
+#create items
+
+@list = List.last
```
Comment (eliotsykes) [inline db/seeds.rb:22]: No need for @instance variable, a `list` local variable would be fine. Try to minimise variable scope where possible.

**Pull request merged**